input/resources/StructureDefinition-profile-patient.json

```diff
@@ -45,6 +45,216 @@
         "isModifier": false
       },
       {
+        "id": "Patient.identifier",
+        "path": "Patient.identifier",
+        "slicing": {
+          "discriminator": [
+            {
+              "type": "value",
+              "path": "system"
+            }
+          ],
+          "rules": "open"
+        }
+      },
+      {
+        "id": "Patient.identifier:SSN",
+        "path": "Patient.identifier",
+        "sliceName": "SSN",
+        "short": "SSN Identifier for this patient",
+        "definition": "SSN identifier for this patient."
+				  
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.code",
+        "path": "Patient.identifier.type.coding.code",
+        "fixedCode": "SB"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.coding.display",
+        "path": "Patient.identifier.type.coding.display",
+        "fixedString": "SSN"
+      },
+      {
+        "id": "Patient.identifier:SSN.type.text",
+        "path": "Patient.identifier.type.text",
+        "fixedString": "Social Security Number"
+      },
+      {
+        "id": "Patient.identifier:SSN.system",
+        "path": "Patient.identifier.system",
+        "fixedUri": "http://hl7.org/fhir/sid/us-ssn"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID",
+        "path": "Patient.identifier",
+        "sliceName": "MedicaidID",
+        "short": "Medicaid identifier for this patient",
+        "definition": "Medicaid identifier for this patient."
+				  
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.coding.system",
+        "path": "Patient.identifier.type.coding.system",
+        "fixedUri": "https://www.hl7.org/fhir/valueset-identifier-type.html"
+      },
+      {
+        "id": "Patient.identifier:MedicaidID.type.coding.code",

```

Comment:
Was reading this https://jira.hl7.org/browse/FHIR-27538, chaange MedicaidID and Medicare ID:
type.coding.code = SB
type.coding.display = Social Beneficiary Identifier

Reply:
Changed